db/package.js

```diff
@@ -116,7 +116,10 @@ class PackageDbController {
       _id: ObjectId(packageId)
     }, {
       $set: {
-        ...(maintainers && { maintainers })
+        ...(maintainers && {
+          maintainers,
+          hasMaintainers: true
```

Comment:
hasMaintainers: !!maintainers.length

Reply:
just in case for some reason we update maintainers: [] in the future, we want to set hasMaintainers to false in that case